yjit/src/codegen.rs

```
@@ -5095,7 +5095,11 @@ fn gen_send_cfunc(
         cme,
         recv,
         sp,
-        pc: Some(0),
+        pc: if cfg!(debug_assertions) {
+            Some(!0) // Poison value. Helps to fast fast.
```

What does "fast fast" mean? Did you mean something like "detect fast"?

---

Sorry, I meant "fail fast":  b235552